typewriter/fixes/fixer_utils.py

```diff
@@ -0,0 +1,120 @@
+"""
+Typewriter specific fixer utility functions
+"""
+
+from lib2to3.fixer_util import (FromImport, Leaf, Newline, Node,
+                                does_tree_import, find_root, is_import, syms,
+                                token)
+
+
+def type_by_import_stmt(package, name, node):
+    """
+    Return name that needs to be annotated based on available imports
+
+    Parameters
+    -------------
+    package : str
+    name : str
+    node : Node
+
+    Returns
+    -------------
+    Optional[str]
+        Returns valid name for type if current import statement exists, or None is there is no import statement
+    """
+    root = find_root(node)
+
+    if does_tree_import(package, name, root):
+        # name is being imported directly in the form of
+        # from <package> import <name>
+        return name
+
+    if does_tree_import(None, package, root):
+        # Mod is being imported in the form of
+        # import <package>
+        return '.'.join((package, name))
+
+    return  # return None to signal there is not yet an import statement
+
+
+def create_import(package, name, node):
+    """
+    Create import statement for type <name> from <package>
```

Comment:
what form of import does this create?

Can we reword this as:

```
Create import statement of the form `from <package> import <name>`
```

or whatever is correct

Reply:
done